src/torch_ppr/utils.py

```diff
@@ -187,8 +197,9 @@ def prepare_page_rank_adjacency(
     adj = edge_index_to_sparse_matrix(edge_index=edge_index, num_nodes=num_nodes)
     # symmetrize
     adj = adj + adj.t()
+    # TODO: should we add an identity matrix here?
     # adjacency normalization: normalize to col-sum = 1
-    degree_inv = torch.reciprocal(torch.sparse.sum(adj, dim=0).to_dense())
+    degree_inv = torch.reciprocal(torch.sparse.sum(adj, dim=0).to_dense().clamp_min(min=1.0e-08))
```

Comment:
better to use the system epsilon here?

Reply:
https://github.com/mberr/torch-ppr/pull/18/commits/d66e61af0c90198f5b5fc2be1b5ee34d22280a6f